cryptoshred-core/src/main/java/eu/prismacapacity/cryptoshred/core/CryptoContainer.java

```diff
@@ -57,8 +55,7 @@
 		private JavaType contextualType;
 
 		@Override
-		public CryptoContainer<?> deserialize(JsonParser jp, DeserializationContext ctxt)
-				throws IOException, JsonProcessingException {
+		public CryptoContainer<?> deserialize(JsonParser jp, DeserializationContext ctxt) throws IOException {
```

Comment:
looks like we're using different formatting standards here (line length)

Reply:
hm, I was  only using spotless to auto-format on commit. I think the new version is just shorter so it removed the extra line.

Reply:
there are no spotless rules in this project, so... hence #134

Reply:
Is this better for you ? :
```suggestion
		public CryptoContainer<?> deserialize(JsonParser jp, DeserializationContext ctxt)
		                throws IOException {
```